guides/operations/setup/installing_sqream_with_monit.rst

```diff
@@ -3,6 +3,8 @@
 *********************************************
 Installing SQream with Monit
 *********************************************
+Monit is an Open Source utility for managing and monitoring Unix systems and conducting automatic maintenance and repair. For example, it can be used for executing meaningful causal actions in error situations, such as starting processes that are not running.
+
 This procedure describes how to manually install SQream with Monit using the **systemd** initialization system.
```

Comment:
This procedure describes how to manually install SQream and optionally launch the instances using Monit.